Formula/cryfs.rb

```diff
@@ -43,12 +43,22 @@ class Cryfs < Formula
     sha256 "9f0e9a2cb5ef521cbfd104d43a208dd9124dfb4accfa72d694e0d0430a0142bc"
   end
 
+  # Fix build with fmt 9+
+  # https://github.com/cryfs/cryfs/pull/433
+  patch do
+    url "https://github.com/cryfs/cryfs/commit/01cf1d5fc98b6c9ac4d7dacb59c6fb787225ea48.patch?full_index=1"
+    sha256 "1ad5022b6054e9ee98721c30cd8c038bf5f2fb5750047a954df62aefbd1ee3fd"
+  end
+
   def install
-    venv = virtualenv_create(buildpath/"venv", "python3")
+    python = "python3.10"
+    venv_root = buildpath/"venv"
+
+    venv = virtualenv_create(venv_root, python)
     venv.pip_install resource("versioneer")
-    xy = Language::Python.major_minor_version "python3"
-    ENV.prepend_path "PYTHONPATH", buildpath/"venv/lib/python#{xy}/site-packages"
-    ENV.prepend_path "PATH", buildpath/"venv/bin"
+
+    ENV.prepend_path "PYTHONPATH", venv_root/Language::Python.site_packages(python)
+    ENV.prepend_path "PATH", venv_root/"bin"
```

Comment:
Not sure why `PYTHONPATH` is needed, since any `python` used will be the one in the `venv`.